report_custom_filename/controllers/report_controller.py

```diff
@@ -19,7 +19,7 @@
 #
 ##############################################################################
```

Comment:
Please change all headers to the [new shorter version](https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3)